dymos/phase/phase.py

```diff
@@ -639,8 +631,8 @@ def add_control(self, name, units=_unspecified, desc=_unspecified, opt=_unspecif
                                  rate2_continuity_scaler=rate2_continuity_scaler,
                                  rate2_continuity_ref=rate2_continuity_ref)
 
-    def set_control_options(self, name, units=_unspecified, desc=_unspecified, opt=_unspecified,
-                            fix_initial=_unspecified, fix_final=_unspecified, targets=_unspecified,
+    def set_control_options(self, name, control_type=_unspecified, order=_unspecified, units=_unspecified, desc=_unspecified,
```

Comment:
Same as above, move control_type and order to the end, and deprecate `set_polynomial_control_options`.

Reply:
I've added it back in and added a deprecation warning.